drivers/i2s/i2s_nrfx.c

```diff
@@ -95,6 +95,26 @@ static void find_suitable_clock(const struct i2s_nrfx_drv_cfg *drv_cfg,
 	nrf_i2s_mck_t best_mck_cfg = 0;
 	uint32_t best_mck = 0;
 
+#if defined(CONFIG_I2S_NRFX_ALLOW_MCK_BYPASS) && NRF_I2S_HAS_CLKCONFIG
+	/* Check for bypass before calculating f_MCK*/
+	for (r = 0; r < ARRAY_SIZE(ratios); ++r) {
+		if (i2s_cfg->frame_clk_freq * ratios[r].ratio_val == src_freq) {
+			LOG_INF("MCK bypass calculated");
+			best_r = r;
+			best_mck = src_freq;
+			best_diff = 0;
+
+			best_mck_cfg = NRF_I2S_MCK_32MDIV8;
```

Comment:
What's the point in setting this? The value is written to the `CONFIG.MCKFREQ` register, which is ignored when the bypass is enabled.